frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/EmbedCodePane.jsx

```diff
@@ -7,13 +7,26 @@ import {
   getSignTokenOptions,
 } from "metabase/public/lib/code";
 
-import CodeSample from "./CodeSample";
-
 import "ace/mode-clojure";
 import "ace/mode-javascript";
 import "ace/mode-ruby";
 import "ace/mode-html";
 import "ace/mode-jsx";
+import { trackStaticEmbedCodeCopied } from "metabase/public/lib/analytics";
+import { EMBED_MODAL_TABS } from "metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane";
+import CodeSample from "./CodeSample";
+
+const getEmbedModalTabLocation = tab => {
```

Comment:
Uses the locations that I mentioned earlier to get the right property enum for the snowplow event. Maybe we can throw this into `trackStaticEmbedCodeCopied` or as a helper function within that snowplow tracking file

Reply:
if we ask product to rename `code_params` to `code_parameters` this could become `code_${tab}`, WDYT?